### PR TITLE
New modules

### PR DESCRIPTION
**Pull request by ChristinaLK**
Licensed software guide mostly done; need to do MPI guide next.

### CODE REVIEW

**Comment by ChristinaLK** [comment]
cc @tjslauson

**Comment by ChristinaLK** [comment]
@jmvera255 where is that note in the MPI guide?

**Comment by jmvera255** [comment]
@ChristinaLK https://github.com/CHTC/chtc-website-source/blob/63394b040157adbb37efb6a8ec4bf2f1c39a01f5/mpi-jobs.shtml#L183-L184

Lines 183-184 of MPI

**Comment by ChristinaLK** [comment]
Ah, that's commented out. `<!-- -->` are html comments. I should probably just delete because it's no longer accurate.

File: licensed-software.shtml [hunk 1]
```diff
@@ -122,11 +138,14 @@ a submit file for jobs that use our licensed software modules.
 They are shown in the sample submit file below and include: </p>
 
 <ul>
-	<li><b>Require access to the modules.</b> There is a special requirements statement
-	that ensures a computer has access to Gluster (where the modules live) and 
-	that the module command is working. </li>
+	<li><b>Require access to the modules.</b> To ensure that your job will have access 
+	to CHTC software modules you must include the following in your submit file.
+	<pre class="sub">
+requirements = (HasCHTCSoftware == true)</pre></li>
```
Comment (tjslauson) [inline licensed-software.shtml]: This attribute is called "HasChtcSoftware" (case difference) in the example job below.  Match the two?

File: mpi-jobs.shtml [hunk 2]
```diff
@@ -149,11 +170,16 @@ a submit file for multicore jobs.  They are shown in the sample submit
 file below and include: </p>
 
 <ul>
-	<li><b>Require access to MPI modules.</b> There is a special requirements statement
-	that ensures a computer has access to Gluster (where the MPI modules live) and 
-	that the module command is working. </li>
+	<li><b>Require access to MPI modules.</b> To ensure that your job will have access 
+	to CHTC software modules, including MPI modules, you must include the following in your submit file. 
+	<pre class="sub">
+requirements = (HasCHTCSoftware == true)
```
Comment (tjslauson) [inline mpi-jobs.shtml]: This attribute is called "HasChtcSoftware" (case difference) in the example job below.  Match the two?

File: mpi-jobs.shtml [hunk 3]
```diff
@@ -186,11 +212,14 @@ executable = <i>run_mpi</i>.sh
...
+transfer_input_files = (this should be a comma separate list of input files if needed)
 
-## Required options for accessing modules
-requirements = ( Target.HasModules == true )
 getenv = true
+# Requirement for accessing new set of modules
+requirements = ( Target.HasChtcSoftware == true ) 
+# Old requirement for previous set of MPI modules
+# These modules will not be available after April 20
+# requirements = ( Target.HasModules == true )
```
Comment (tjslauson) [inline mpi-jobs.shtml]: licensed_software.shtml doesn't prefix these attributes with Target.  I'm forgetting right now which syntax is correct (or more correct), but they should be unified.

Comment (ChristinaLK) [reply mpi-jobs.shtml]: Technically `Target.` is more correct but I don't think we need it + we don't use it for other requirements. I'll remove.

**Comment by jmvera255** [comment]
using "Target." would make it consistent with other requirements statements on our page

**Pull request merged**